diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -503,7 +504,8 @@ bool DiffDriveController::reset()
   subscriber_is_active_ = false;
   velocity_command_subscriber_.reset();
 
-  received_velocity_msg_ptr_.set(nullptr);
+  received_velocity_msg_ptr_.set([](std::shared_ptr<TwistStamped> & stored_value)
+                                 { stored_value = nullptr; });
```

Comment:
With latest fixes to realtime_tools this could stay like
```suggestion
  received_velocity_msg_ptr_.set(nullptr);
```
as the reviewers prefer.